src/Integration/Flysystem/S3/composer.json

```diff
@@ -22,10 +22,10 @@
         "php": "^7.2.5 || ^8.0",
         "async-aws/s3": "^1.5",
         "league/flysystem": "^1.0.65",
-        "league/mime-type-detection": "^1.0"
+        "league/mime-type-detection": "^1.5.1"
```

Comment:
really? We are not compatible with previous version?

Reply:
I just bumped it to the latest. We are compatible with 1.0. Should I revert?

Reply:
Yes please, If we are compatible with 1.0, we should force user to upgrade to 1.5.1

Reply:
Okey. Thank you